### ISSUE TITLE
Plan

### ISSUE THREAD

**Issue by mshoward** [mshoward/APA#3]
We need to plan everything out before we write any code at all.  This project is a pretty big deal, and I don't think anyone wants to not do well on it.

**Comment by lowrycameron** [comment]
I agree.  I think we are in the right track with being where we are.  Let's
meet Tuesday and start working on it.  This weekend we can all look at
various aspects and then get to work on the actual code.
